etc/ReplayOfflineConfiguration.py

```diff
@@ -104,23 +104,24 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_12_4_10"
+    'default': "CMSSW_12_5_1"
 }
 
 # Configure ScramArch
 setDefaultScramArch(tier0Config, "el8_amd64_gcc10")
 
 # Configure scenarios
-ppScenario = "ppEra_Run3"
-ppScenarioB0T = "ppEra_Run3"
+ppScenario = "ppEra_Run3_pp_on_PbPb"
+ppScenarioB0T = "ppEra_Run3_pp_on_PbPb"
```

Comment:
```suggestion
ppScenario = "ppEra_Run3"
ppScenarioB0T = "ppEra_Run3"
```
The standard scenarios probably should not be changed with the PbPb ones